systemverilog-plugin/UhdmAst.cc

```diff
@@ -1957,39 +1946,14 @@ void UhdmAst::process_module()
                         log_assert(node->children[0]->type == AST::AST_CONSTANT || node->children[0]->type == AST::AST_REALVALUE);
                     }
                 }
-                auto parent_node = std::find_if(module_node->children.begin(), module_node->children.end(), [&](AST::AstNode *child) -> bool {
-                    return ((child->type == AST::AST_PARAMETER) || (child->type == AST::AST_LOCALPARAM)) && child->str == node->str &&
-                           // skip real parameters as they are currently not working: https://github.com/alainmarcel/Surelog/issues/1035
-                           child->type != AST::AST_REALVALUE;
-                });
```

Comment:
Why don't we want to do this part anymore?

Reply:
This part were selecting all parameters that had real value in parameter and was forcing to use ``AST_PARASET`` instead of replacing actual value of parameter as surelog used to report wrong values for real type. This is no longer the case.

I've updated test case to showcase this: https://github.com/chipsalliance/UHDM-integration-tests/pull/717/files#diff-3a52e89ebb9bcd325d69c85bf56ee916ac4dc9990e60241e52d5bd0b14ef89afR20

Reply:
Got it, thanks!